### PR TITLE
build-bottle-pr: check existence of the required remote repository

### PR DESCRIPTION
**Pull request by maxim-belkin**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Linuxbrew/brew/pulls) for the same change?

---

make sure that we have remote repositories before we attempt to build bottles

### CODE REVIEW

File: Library/Homebrew/dev-cmd/build-bottle-pr.rb [hunk 1]
```diff
@@ -27,6 +27,9 @@ def limit
 
   def build_bottle(formula)
     remote = ARGV.value("remote") || ENV["GITHUB_USER"] || ENV["USER"]
+    ohai "Using #{remote} remote to submit Pull Requests"
```
Comment (sjackman) [inline Library/Homebrew/dev-cmd/build-bottle-pr.rb]: Please add `if verbose?` (or whatever the correct code is)

**Review by sjackman — CHANGES_REQUESTED** [review]

File: Library/Homebrew/dev-cmd/build-bottle-pr.rb [hunk 2]
```diff
@@ -27,6 +29,9 @@ def limit
 
   def build_bottle(formula)
     remote = ARGV.value("remote") || ENV["GITHUB_USER"] || ENV["USER"]
+    ohai "Using #{remote} remote to submit Pull Requests" if ARGV.verbose?
```
Comment (maxim-belkin) [inline Library/Homebrew/dev-cmd/build-bottle-pr.rb:32]: added `if ARGV.verbose?` here

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
Looks good. Thanks, Maxim.

**Comment by maxim-belkin** [comment]
So, I'll squash and merge then!

**Pull request merged**